README.md

```diff
@@ -345,6 +359,44 @@ Image flip options.
 import { Flip } from "pi-camera-connect";
 ```
 
+## `ExposureMode`
+Exposure mode options.
+- `ExposureMode.Off`
```

Comment:
I don't see `off` as an exposure mode option in the docs. Can you please confirm that this exists?

Reply:
It's in the CLI help, and I am using this option for my usecase, and it seems to work…

```bash
$ raspistill --help

--snipped--

-ex, --exposure	: Set exposure mode (see Notes)

--snipped--

Notes

Exposure mode options :
off,auto,night,nightpreview,backlight,spotlight,sports,snow,beach,verylong,fixedfps,antishake,fireworks

--snipped--
```

I've never had success using the `off` option for _white balance_, though (`--awb`)… everything looks green when I do that.

Reply:
Okay thanks for confirming. Looks like their online docs are a bit out of sync...

https://www.raspberrypi.org/documentation/raspbian/applications/camera.md

I'll make a note to reference the CLI going forward.